src/datastore/postgres-store.ts

```diff
@@ -5587,6 +5633,7 @@ export class PgDataStore
           UNION ALL (
             SELECT DISTINCT ON (fully_qualified_subdomain) fully_qualified_subdomain as name, zonefile
             FROM subdomains
+            LEFT JOIN zonefiles ON zonefiles.zonefile_hash = subdomains.zonefile_hash
```

Comment:
Can you test how this change impacts the execution time of this query on the full BNS dataset? It's not clear to me if postgres will continue to use the sql index that makes this query fast enough. Related to the comment here: https://github.com/blockstack/stacks-blockchain-api/pull/730#discussion_r700394859

cc @asimm241 who might be able to help

Reply:
With previous query (for same table for zonefiles and names):
Planning time: 0.860 ms
Execution time: 0.436 ms
Total time: 1.296 ms

With this query:
Planning Time: 10.892 ms
Execution Time: 1.321 ms
Total Time: 12.213 ms

@zone117x

Reply:
Thanks for the update @MHassanTariq. Can you confirm the query was done on the full BNS subdomain dataset, which should be something like 2 million rows? Also, were able to test the other modified query (inside `getSubdomain`)?

Reply:
Also, while the total time of both previous and new query appear low, the relative difference between the 2 suggests the JOIN is not particularly optimal. Ideally we be seeing a new total time of: `original query time, e.g. 1.3ms` + `query time to lookup a zonefile by hash (should be faster than first query)`.

Apologies if this seems nitpicky, but pg query performance is something we're becoming more stringent about.

Reply:
@zone117x regarding the `getSubdomain` query, it was taking total time for execution of `18.539ms` and after dividing names and zonefiles into two tables, it is now taking a total time of `15.040ms`.

Your exact suggested solution was proposed by @asimm241 too. I analyzed with that and got a total time of `3.759 ms` for `getLatestZonefile`, I think I should change my approach to the new one. 
1. Getting the zonefile_hash from the first query gives the total time of `3.366ms`
2. Fetching zonefile from zonefile_hash takes `0.393ms`.

Yep, all times were calculated with the full BNS subdomain dataset having more than 2 million rows.